components/basecomponents/SearchInput/index.ts

```diff
@@ -0,0 +1 @@
+export { default as SearchInput } from './SearchInput';
```

Comment:
What is the purpose of this file?

Reply:
Move component to own folder, so there could be tests, helpers, etc. if needed in the folder:

import { SearchInput } from 'components/basecomponents/SearchInput'
vs.
import { SearchInput } from 'components/basecomponents/SearchInput/SearchInput'

If not desired in this project, I will revert the changes.

Reply:
Yeah, I see how that can be useful for large projects, but we are not there yet. Also having many `index.ts` files for each page is already confusing, having them for components as well would make that worse imo. I would try to avoid unnecessary folder nesting until it's really necessary :)